tests/ui/rfcs/rfc-2632-const-trait-impl/effects/minicore.rs

```diff
@@ -398,17 +391,16 @@ impl<'a, T: ?Sized> Pin<&'a T> {
     }
 }
 
-
 impl<P: Deref> Pin<P> {
-    /* const */ fn as_ref(&self) -> Pin<&P::Target>
+    /* const */
+    fn as_ref(&self) -> Pin<&P::Target>
     where
-        P: /* ~const */ Deref,
```

Comment:
pls unformat this code, it deleted a bunch of comments

Reply:
whoops